addons/payment_stripe/tests/test_stripe.py

```diff
@@ -76,3 +77,33 @@ def test_30_stripe_form_management(self):
         tx.form_feedback(stripe_post_data, 'stripe')
         self.assertEqual(tx.state, 'done', 'Stripe: validation did not put tx into done state')
         self.assertEqual(tx.acquirer_reference, stripe_post_data.get('id'), 'Stripe: validation did not update tx id')
+
+    def test_add_available_payment_method_types(self):
+        currency_mock = Mock()
+        currency_mock.name = 'eur'  # name is a keyword in the constructor, we need to override it this way
+        tx_values = {
+            'billing_partner_country': Mock(code='be'),
```

Comment:
you could have used base data here:`self.env.ref('base.be')` (considered part of the base installation, so can be relied on)

Reply:
same for currency btw

Reply:
Indeed.

**I used `self.env.ref`.**